functions/src/index.ts

```diff
@@ -0,0 +1,81 @@
+import admin from "firebase-admin";
+import functions from "firebase-functions";
+
+// TODO: do we need to import firebase here like this, or can we just use it from firebase-admin?
+import firebase from "firebase/app";
+import "firebase/firestore";
```

Comment:
[TODO] do we need to import firebase here like this, or can we just use it from `firebase-admin`?